client/src/IconClicked.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+
+export function IconClicked() {
+  return (
+    <>
+      <div className="flex-container">
+        <Link to="/movieApp" className="icon-text">
+          FEED
+        </Link>
+        <Link to="/movieApp/RECOMMENDATION" className="icon-text">
```

Comment:
Since the IconClicked component is rendered inside the `/movieApp` route, you can use relative paths here. That makes it a little easier to maintain:
`<Link to="RECOMMENDATION" className="icon-text">`

Note: I can't find something definitive on what to do for your FEED link. I would try either `to=""` or `to="."`. If neither of those work, you can add a Route in your App that is the same as the `index` route but with `FEED` as the path:
```
          <Route index element={<FeedComponent />} />
          <Route path="feed" element={<FeedComponent />} />
```
then you could link `to="FEED"`.